db/seeds/application_types.yml

```diff
@@ -24,6 +24,14 @@
     - project_id_service_account_json
     openshift:
     - token
+"/insights/platform/rh-marketplace":
```

Comment:
I am not sure if we need any new application type.

I guess we are going to just create an source type of red hat marketplace with a token and other applications could connect to that. 🤔 

But it’s a totally new case for Sources so I am not really sure.

Reply:
I am betting you're right on this as well. Let me ping @syncrou and @lindgrenj6 on this just to confirm it but for now I'm going to remove that part.